src/gui/folderman.h

```diff
@@ -277,9 +280,6 @@ private slots:
     /* unloads a folder object, does not delete it */
     void unloadFolder(Folder *);
 
-    /** Will start a sync after a bit of delay. */
-    void startScheduledSyncSoon();
```

Comment:
Why did you move this? :-)
Use ```git show``` before pushing to see the commit if it makes sense.
You can use ```git commit --amend``` to change your existing commit.

Reply:
It was private and needed to be public. Should be undone when the Account reconnection is used instead.

Reply:
@ckamm Meh, github didn't display the context properly. I thought he changed it from "private **slots**" to not being a slot.

Reply:
@ckamm Indeed I will undo that.

@guruz Thank you for you help. :)
Unfortunately git commit --amend dont work because the last commit I deleted some comments but I allready changed the postion.